### DISCUSSION TITLE
Extend feature for /tn for personalised list columns to Report Builder

### DISCUSSION THREAD

**Discussion by VanWeapon** [arnoudkooi/SN-Utils#298] [Ideas]
When using Show Technical Names feature in the Personalise List Columns dialog, you get a bunch of extra features including adding hidden/dot walked fields to the list. 

Requesting to extend this to the Report Builder UI for List reports. Normally you can't add the sys_id fields to reports, and this would help with that. Also the feature for filtering column names would be nice.

It is possible to get around this by manually editing the `field_list` field of a sys_report record, but this has to be done via a background script as the field is not normally editable. This would be cleaner

**Comment by arnoudkooi** [comment]
This should be in 6.1.4.4, thanks!

**Comment by VanWeapon** [comment]
Works like a charm
